src/_guides/language/language-tour.md

```diff
@@ -2237,24 +2237,23 @@ callbacks.forEach((c) => c());
 The output is `0` and then `1`, as expected. In contrast, the example
 would print `2` and then `2` in JavaScript.
 
-If the object that you are iterating over is an Iterable, you can use the
-[forEach()][] method. Using `forEach()` is a good option if you don’t need to
-know the current iteration counter:
+If the object that you are iterating over is an Iterable (such as List or Set),
+you can use the `for-in` form of [iteration](/guides/libraries/library-tour#iteration)
+if you don't need to know the current iteration counter:
```

Comment:
Now that you made this one sentence, it seems a bit strange to have the two conditions separated here. Perhaps combining the them with `and` or keeping them as separate sentences like before.

I'm leaning a bit towards keeping the conditional sentences separate as you technically "can" use `for-in` even if you do need the current iteration counter or if an iteration counter doesn't even make sense for the collection type.

Reply:
I'll combine them with "and".  If someone needs an iteration counter, they should prefer `for` instead of `for-in`.